src/table/utils/__tests__/styling-utils.spec.js

```diff
@@ -113,6 +113,29 @@ describe('styling-utils', () => {
         padding: '6px 12px',
       });
     });
+    it('should return styling with fontSize and padding when the index for font color is -1 and color is null', () => {
+      styleObj.fontColor = { index: -1, color: null };
+      const resultStyling = getBaseStyling(styleObj, objetName, theme);
+      expect(resultStyling).toEqual({
+        backgroundColor: '#323232',
+        borderColor: '#D9D9D9',
+        borderStyle: 'solid',
+        fontSize: 12,
+        padding: '6px 12px',
```

Comment:
Is this the real case scenario? that you don't get a color set?

Reply:
only if there is something set on the theme?

Reply:
You are right. When there is no color from Layout and color from the theme, you get color from the theme. 
In the test case, there is no color from either Layout or theme, so you don't get any color.

Reply:
Adding a new test case addressing it.